mkdocs.yml

```
@@ -49,6 +49,9 @@ nav:
 - Guidelines:
   - Contributing: guidelines/contributing.md
   - Documentation: guidelines/docs.md
+- How-to:
+  - Add:
+    - A Go module: how-to/add/go-module.md
```

what do you think about this?

```suggestion
  - Go:
    - Add module: how-to/go/add-module.md
```

---

I like it! Fixed in [0fad114](https://github.com/DataDog/datadog-agent/pull/27196/commits/0fad114fc520cb3a9a2a583c1b1c7e3d6e3dd7e8)!